core/src/main/java/org/fao/geonet/kernel/search/EsSearchManager.java

```diff
@@ -151,6 +151,11 @@ public class EsSearchManager implements ISearchManager {
             .add(Geonet.IndexFieldNames.RESOURCEABSTRACT + "Object")
             .add("operatesOn")
             .build();
+
+        FIELDLIST_RELATED_SCRIPTED = ImmutableMap.<String, String>builder()
+            // ElasticSearch scripted field to get the first overview url. Scripted fields must return single values.
```

Comment:
The only drawback here is that we don't benefit from small index preview and client app will have to load all images

https://sdi.eea.europa.eu/catalogue/srv/eng/catalog.search#/metadata/3c362237-daa4-45e2-8c16-aaadfb1a003b

but it'll be way faster when browsing child records

https://sdi.eea.europa.eu/catalogue/srv/eng/catalog.search#/metadata/59ed0aa1-a464-446b-a960-36df73e2a803f